app/src/main/java/com/android/unio/ui/authentication/Welcome.kt

```diff
@@ -53,14 +54,17 @@ import com.google.firebase.Firebase
 import com.google.firebase.auth.auth
 
 @Composable
-fun WelcomeScreen(navigationAction: NavigationAction) {
+fun WelcomeScreen(
+    navigationActions: NavigationAction,
```

Comment:
Consider removing both these parameters as they are not used by the function.

Reply:
Thanks forgot about these